### PR TITLE
create yt espresso series tweet

### PR DESCRIPTION
**Pull request by oindrillac**
first tweet 🥇 

@durandom please suggest any changes

### CODE REVIEW

File: tweets/2021-04/yt-espresso.tweet [hunk 1]
```diff
@@ -0,0 +1 @@
+Operate First beings users, developers and operators closer together! Check out https://www.youtube.com/playlist?list=PL8VBRDTElCWpneB4dBu4u1kHElZVWfAwW, our new YouTube Espresso series to learn what Operate First is all about and learn about how we build software and perform data science in the open through short videos.
```
Comment (durandom) [inline tweets/2021-04/yt-espresso.tweet:1]: Operate First brings users, developers and operators closer together! Check out our new YouTube Espresso series to learn about Operate First and how we perform data science in a cloud-native way. https://www.youtube.com/playlist?list=PL8VBRDTElCWpneB4dBu4u1kHElZVWfAwW

**Review by durandom — CHANGES_REQUESTED** [review]

File: tweets/2021-04/yt-espresso.tweet (hunk 1 above)
Comment (oindrillac) [reply tweets/2021-04/yt-espresso.tweet]: @durandom changes made 🙂

**Review by durandom — APPROVED** [review]
awesome, :shipit:  ❤️

**Comment by oindrillac** [comment]
@durandom sesheta is not working on this repo now. @tumido is investigating the issue

**Pull request merged**